python/ray/experimental/gpu_object_manager/gpu_object_store.py

```diff
@@ -144,15 +147,23 @@ def __init__(self):
 
     def has_object(self, obj_id: str) -> bool:
         with self._lock:
-            return obj_id in self._gpu_object_store
```

Comment:
can you add the assertion here?

```python
existed = obj_id in self._gpu_object_store
if existed:
  assert len(self._gpu_object_store[obj_id]) > 0
return existed
```

Then, use `has_object` to replace the queue size checks in the file.